server/lib/schema/index.ts

```diff
@@ -2094,6 +2094,64 @@ const schema = new GraphQLSchema({
           },
         ),
       },
+      updateTreeExpertAssignments: {
+        type: GraphQLBoolean,
+        args: {
+          treeId: { type: GraphQLString },
+          honestEmails: { type: GraphQLList(GraphQLString) },
```

Comment:
To maintain consistency with the other API endpoint, we should probably use `emailsOfHonestExperts` and `emailsOfMaliciousExperts` as the names for these arguments.

Reply:
Noticing that we’re slowly migrating from `Oracle` to `Expert` terminology in the codebase. I’m on the fence about just renaming everything `Expert`, as I feel like code like this (unrelated to PR) could be confusing. 
![image](https://user-images.githubusercontent.com/1892071/64042356-86098280-cb16-11e9-900e-ff092370a11f.png)
My feeling is that with Mosaic2 on the way, it's probably not worth the time. We should just use Oracle term in the codebase. The exception being for user-facing language.

Reply:
Yeah I see where you're coming from. Ever since we decided to use "expert" instead of "oracle", I've used "expert" when creating a new variable/field, e.g., `isAwaitingHonestExpertDecision`. I do think there's an argument that I should have just kept using "oracle", though.